server/app.js

```diff
@@ -3,9 +3,13 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const connectDB = require('./config/dbtest')
```

Comment:
You're assigning a const and reassigning it later, did you mean to remove the connectDB definition later?

Additionally, the name dbtest for the file is slightly misleading, maybe just db would be better

Reply:
Actually dbtest is a temp file I had created until Aidan's work was approved and merged into the project so I can access the DB and run some tests. But makes no sense anymore to keep it indeed. I just removed line 6 alltogether.